packages/astro/src/core/config.ts

```diff
@@ -36,7 +71,19 @@ export const AstroConfigSchema = z.object({
 		.optional()
 		.default('./dist')
 		.transform((val) => new URL(val)),
-	renderers: z.array(z.string()).optional().default(['@astrojs/renderer-svelte', '@astrojs/renderer-vue', '@astrojs/renderer-react', '@astrojs/renderer-preact']),
+	integrations: z.array(z.any()).default([]),
```

Comment:
Glad to see this defaults to none!

Reply:
Yea... are we okay with this? It makes it a more disruptive change if so. I might ping you all on Discord to confirm that this is what we want, since we don't need to make this breaking change now. 

But like all things, maybe best to just rip the bandaid off now

Reply:
Oh wow, I had no clue it defaulted to _all_ frameworks previously. I definitely think we'll move renderers out of the core and into separate packages down the road. The requirement to install every supported UI framework makes `npm init astro` + plain npm (no yarn or pnpm) a pain to use. As a stepping stone to this, I'd say we should default to none for 1.0.